packages/client/hmi-client/src/components/workflow/scenario-templates/horizon-scanning/tera-horizon-scanning-template.vue

```diff
@@ -0,0 +1,238 @@
+				option-value="id"
+				@update:model-value="scenario.setModelConfigSpec($event)"
+				:disabled="isEmpty(modelConfigurations) || isFetchingModelInformation"
+				:loading="isFetchingModelInformation"
+			/>
+			<label>Select uncertain parameters of interest and adjust ranges to be explored if needed</label>
+			<template v-for="(parameter, i) in scenario.parameters" :key="i">
+				<div class="flex">
+					<Dropdown
+						class="flex-1"
+						:model-value="parameter?.id"
+						:options="modelParameters"
+						option-label="referenceId"
+						option-value="referenceId"
+						placeholder="Select a parameter"
+						:disabled="!selectedModelConfiguration"
+						:loading="isFetchingModelConfiguration || isFetchingModelInformation"
+						@update:model-value="onParameterSelect($event, i)"
+					>
+						<template #option="slotProps">
+							<span>{{ displayParameter(slotProps.option.referenceId) }}</span>
+						</template>
+
+						<template #value="slotProps">
+							<span v-if="displayParameter(slotProps.value)">{{ displayParameter(slotProps.value) }}</span>
+							<span v-else>{{ slotProps.placeholder }}</span>
+						</template>
+					</Dropdown>
+					<Button v-if="scenario.parameters.length > 1" icon="pi pi-trash" text @click="scenario.removeParameter(i)" />
+				</div>
+				<div v-if="parameter" class="distribution-container">
+					<label class="p-0 white-space-nowrap">Extreme low:</label>
+					<tera-input-number class="m-0" v-model="parameter.low" auto-width />
+					<label class="p-0 ml-2 white-space-nowrap">Extreme high:</label>
+					<tera-input-number class="m-0" v-model="parameter.high" auto-width />
+				</div>
+			</template>
+			<div>
+				<Button
+					:disabled="!scenario.modelSpec.id"
+					class="mt-2"
+					label="Add parameter"
+					icon="pi pi-plus"
+					text
+					size="small"
+					@click="scenario.addParameter()"
+				/>
+			</div>
+		</template>
+		<template #outputs>
+			<label>Select an output metric</label>
+			<MultiSelect
+				:disabled="isEmpty(modelStateOptions) || isFetchingModelInformation"
+				:model-value="scenario.simulateSpec.ids"
+				placeholder="Select output metrics"
+				option-label="name"
+				option-value="id"
+				:options="modelStateOptions"
+				@update:model-value="scenario.setCalibrateSpec($event)"
+				:loading="isFetchingModelInformation"
+				filter
+			/>
+			<!-- <img :src="simulate" alt="Simulate chart" /> -->
+		</template>
+	</tera-scenario-template>
+</template>
+
+<script setup lang="ts">
+import { HorizonScanningScenario } from '@/components/workflow/scenario-templates/horizon-scanning/horizon-scanning-scenario';
+import { useProjects } from '@/composables/project';
+import { getInterventionPoliciesForModel, getModel, getModelConfigurationsForModel } from '@/services/model';
+import { AssetType, InterventionPolicy, ModelConfiguration, ParameterSemantic } from '@/types/Types';
+import _, { isEmpty } from 'lodash';
+import { computed, ref, watch } from 'vue';
```

Comment:
nitpick: preferable to have the imports in the following order:
- lodash and other vendor utilities
- vue stuff
- our internal services/components ... etc

This ordering makes things a little safer in case any library decides to override stuff, eg us override lodash or whatnot.

Reply:
good to know, I've reorganized the imports here for the horizon scanning